miniapps/gslib/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@
 # CONTRIBUTING.md for details.
 
 if (MFEM_USE_GSLIB)
+  add_mfem_miniapp(schwarz_ex1
```

Comment:
The names of theses executables aren't consistent with what's outputted by the makefile. Please update these to `ex1` and `ex1p`

Reply:
There are some conflicts with the CMake system if I use `ex1` since there is already an `ex1` in the examples directory. For consistency, I am renaming the examples to `schwarz_ex1` and `schwarz_ex1p`.

Reply:
Thanks for the CMake fixes :)